python/lsst/dax/apdb/tests/data_factory.py

```diff
@@ -135,7 +135,7 @@ def makeSourceCatalog(objects: pandas.DataFrame, visit_time: DateTime,
 
 def makeForcedSourceCatalog(objects: pandas.DataFrame, visit_time: DateTime,
                             ccdVisitId: int = 1) -> pandas.DataFrame:
-    """Make a catalog containing a bunch of DiaFourceSources associated with
+    """Make a catalog containing a bunch of DiaFourcedSources associated with
```

Comment:
is there a "u"? I think it's DiaForcedSources

Reply:
Indeed, pardon my French 🙂